core_backend/app/question_answer/schemas.py

```diff
@@ -37,6 +37,8 @@ class QueryBase(BaseModel):
     )
     query_text: str = Field(..., examples=["What is AAQ?"])
     session_id: int | None = Field(default=None)
+    turnio_api_key: Optional[str] = None
```

Comment:
`turnio_api_key` and `wa_id` are now part of `QueryBase` as optionals. When **both** are specified, then the `chat` endpoint will run in the background. Otherwise, the `chat` endpoint will run async as usual.